hoomd/md/EvaluatorPairExpandedLJ.h

```diff
@@ -38,8 +38,8 @@ namespace md
 
     EvaluatorPairExpandedLJ evaluates the function:
     \f{eqnarray*}
-    V_{\mathrm{SLJ}}(r)  = & 4 \varepsilon \left[ \left( \frac{\sigma}{r - \Delta} \right)^{12} -
```

Comment:
Is this something we missed when reviewing the `ExpandedLJ` PR?

Reply:
Yes